mitodl/open-discussions
# dont start extra tasks for index updates

#### Pre-Flight checklist

#### What are the relevant tickets?
closes https://github.com/mitodl/open-discussions/issues/4004

#### What's this PR do?
We had an issue where an ocw course was deleted and it took most of a day for the change to propagate to the search index.  Also for some of that time the course was already removed from the open database so it came up in /learn/search and /infinite/search but opened to a blank drawer. Not starting separate task for reindexing will improve the speed that things are indexed

#### How should this be manually tested?
Run some of the backpopulate_ tasks in course_catalog/management/ and verify that you don't see errors and the index is updating. 

Make a new post or comment in disscussions and verify that the new post is available in the  /search page
